tests/test_rigid_physics.py

```diff
@@ -2169,6 +2169,37 @@ def test_terrain_size(show_viewer, tol):
     assert_allclose((height_ref * 2.0), height_test, tol=tol)
 
 
+@pytest.mark.required
+@pytest.mark.parametrize("backend", [gs.cpu])
```

Comment:
You can keep the unit test for weld constraints since it is missing, but add back the unit test for the Jacobian that has been removed for some reason.

Reply:
Plus, could you check that welding is working by adding an initial velocity to one of the 2 boxes and check that moving are moving in the same direction with the same speed ? You should disable gravity to make the check easier.